nodes/src/nodes/dataset_cobalt/IGlobal.py

```
@@ -0,0 +1,166 @@
+        bag = self.IEndpoint.endpoint.bag
+        config = self._extractConfig()
+        debug(f'Cobalt Dataset Global: Config keys: {list(config.keys())}')
+
+        # Create loader and load dataset
+        self._loader = DatasetLoader(config, bag)
+
+        try:
+            items = self._loader.load()
+            debug(f'Cobalt Dataset Global: Loaded {len(items)} raw items')
+        except FileNotFoundError as e:
+            warning(f'Cobalt Dataset Global: {str(e)}')
```

Ruff (0.15.10)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)